### DISCUSSION TITLE
EGLOT special buffers unaware of JSON-RPC connection

### DISCUSSION THREAD

**Discussion by doolio** [joaotavora/eglot#1035] [Q&A]
I've noticed the ELOT special buffers such as the events and stderr buffers (are there others?) are unaware of the JSON-RPC connection. Should they not? I noticed this when I tried to open the stderr buffer when in the events buffer and got this error message:

> jsonrpc-error: jsonrpc-error: "No current JSON-RPC connection", (jsonrpc-error-code . 32603), (jsonrpc-error-message . "No current JSON-RPC connection")

**Comment by joaotavora** [comment]
Maybe they could be yes, if that can be done in under 5 loc. Else i don't
think it's worth the complexity.
